include/rest/api_client.hpp

```diff
@@ -144,19 +243,11 @@ namespace disccord
                     }
 
                     template <typename TModel>
-                    pplx::task<void> request_multi_json(route_info& route, std::vector<TModel> body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
+                    pplx::task<void> request_multi_json(route_info& route, TModel body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
```

Comment:
yeah, makes sense. reason why I was thrown off by this is we shouldn't be turning the model into an array with `request_multi_json`, we should expect it to already be an array. merging what I have in my `request_json_array` into `request_multi_json` will do that.